### ISSUE TITLE
MacOS: connect_failed asio.system:2

### ISSUE THREAD

**Issue by caizoryan** [kmonad/kmonad#730]
When I try to run kmonad I get the error 

```
connect_failed asio.system:61
```

```
defaults read /Applications/.Karabiner-VirtualHIDDevice-Manager.app/Contents/Info.plist CFBundleVersion
```
 
outputs

```
1.15.0
``` 

My config is 

```
(defcfg
  input (iokit-name "Apple Internal Keyboard / Trackpad")
  output (kext)
  fallthrough true
  allow-cmd true
)

```

**Comment by slotThe** [comment]
Could you try #716?

**Comment by YosefBayoude** [comment]
can confirm, the doc is actually out of date, v2.1.0 of dext should be used as of the writing of this message

**Comment by slotThe** [comment]
Fancy contributing a doc update? :)

**Comment by YosefBayoude** [comment]
[PR created](https://github.com/kmonad/kmonad/pull/776) ;)

**Comment by slotThe** [comment]
Closed by #776
